server/controllers/users.js

```diff
@@ -101,15 +114,16 @@ export const createUser = async (req, res) => {
 
 export const updateUser = async (req, res) => {
   const { id } = req.params;
-  const { username, email, spotifyUserId, lat, lng } = req.body;
+  const { username, email, spotifyUserId, lat, lng, spotifyRefresh } = req.body;
 
   if (!mongoose.Types.ObjectId.isValid(id))
     return res.status(404).json({ message: `No user with id: ${id}` });
 
   const updatedUser = {
     username,
     email,
-    spotifyUserId: spotifyUserId,
+    spotifyUserId,
+    spotifyRefresh,
     lat: lat,
     lng: lng,
     _id: id,
```

Comment:
Because you did the change of "spotifyUserId: spotifyUserID" to "spotifyUserId" we should do the same for "lat: lat" and "lng: lng"